Source/Core/DolphinQt2/MenuBar.h

```diff
@@ -26,6 +28,9 @@ class MenuBar final : public QMenuBar
   void EmulationStopped();
   void UpdateStateSlotMenu();
   void UpdateToolsMenu(bool emulation_started);
+  void UpdateSelection(QSharedPointer<GameFile> game_file);
```

Comment:
Same: I would appreciate some more consistency in the naming here. In 3 new methods that do similar things: two are active form actions ("update selection", "set readonly mode"), one is passive form action ("recording status changed").